data/collections/protocol-labs-network.yaml

```diff
@@ -8,10 +8,13 @@ projects:
   - canvasxyz
   - ccpgames
   - celestiaorg
+  - chainsafe
```

Comment:
@LaurenSpiegel please remove `chainsafe` -- we already have `forest-chainsafe` included in the PLN collection (see further down). This is to separate Forest from Lodestar (`lodestar-chainsafe`).